DDR4 OC Guide.md

```diff
@@ -144,11 +144,11 @@ You should always test with a variety of stress tests to ensure your overclock i
 ### Thaiphoon Report
 * Note: Thaiphoon is known to guess ICs so it shouldn't be fully trusted. It's highly recommended to check the label on the sticks if possible.
   * See [here](https://www.reddit.com/r/overclocking/comments/ig9d76/thaiphoon_burner_cluelessly_guessing_memory_ics/) for more info.
-* [Single rank 8Gb Hynix CJR](https://i.imgur.com/hbFyKB2.jpg).
-* [Single rank 8Gb Micron Revision E](https://i.imgur.com/3pQjQIG.jpg) (source: Coleh#4297).
+* [Single rank 8 GB Hynix CJR](https://i.imgur.com/hbFyKB2.jpg).
+* [Single rank 8 GB Micron Revision E](https://i.imgur.com/3pQjQIG.jpg) (source: Coleh#4297).
   * [SpecTek](https://www.micron.com/support/spectek-support) ICs are lower binned Micron ICs.
-  * Esoteric note: Many people have started calling this Micron E-die or even just E-die. The former is fine, but the latter can cause confusion as letter-die is typically used for Samsung ICs, i.e. 4Gbit Samsung E-die. Samsung is implied when you say E-die, but as people are calling Micron Rev. E E-die, it'd probably be a good idea to prefix the manufacturer.
-* [Dual rank 8Gb Samsung B-die](https://i.imgur.com/Nqn8s76.jpg).
```

Comment:
These are the correct units

8Gb is the capacity of each chip, not the overall size of the RAM stick

Reply:
Ah, those are correct then. They are still missing the space between the unit and number of measurement. It's a pattern that repeats itself throughout the guide. See also: https://en.wikipedia.org/wiki/Space_%28punctuation%29#Unit_symbols_and_numbers

Reply:
I agree

Push through the changes with the GB changes undone and I'll review again

Reply:
I will address this in a commit tomorrow

Reply:
It has been fixed